sdn_tests/testplans/Stress_Latency_Test_Plan.md

```diff
@@ -0,0 +1,385 @@
+# Overview
+
+This document aims to outline the approach for testing the stress and latency of gNMI GET/SET operations supported by the gNMI protocol as part of GPINs OpenConfig end-to-end testing. 
+
+# Background
+
+With SONiC as the network operating system (NOS) for GPINS, gNMI is responsible for monitoring, streaming telemetry, and configuration management.  Broadcom's Unified Management Framework (UMF) provides gNMI streaming telemetry based on the standard OpenConfig model.
```

Comment:
Remove the second sentence on this line. Also remove "for GPINS" in the first sentence.

Reply:
Done.